app/models/developer.rb

```diff
@@ -90,8 +95,23 @@ def feature!
     touch(:featured_at)
   end
 
+  def new_developer_account?
+    created_at > NEW_ACCOUNT_THRESHOLD.ago
+  end
+
   private
 
+  def changes_indicate_potential_hire?
+    return false unless saved_change_to_search_status? && !new_developer_account?
```

Comment:
Let's drop the second clause. Since we are firing on update, and not on create, I'm fine with getting an extra notification if someone changes their status after signing up.